src/Sylius/Bundle/TaxationBundle/Resources/config/doctrine/model/TaxRate.orm.xml

```diff
@@ -21,7 +21,7 @@
 
         <field name="code" column="code" type="string" unique="true" />
         <field name="name" column="name" type="string" />
-        <field name="amount" column="amount" type="decimal" precision="10" scale="5" />
+        <field name="amount" column="amount" type="decimal" precision="10" scale="5" nullable="true" />
```

Comment:
Why is this change necessary? Do TaxRate without amount exists?

Reply:
Yeah I went a bit too easy on this one. Reverted, and treating removing/null values for amount as 0, since that's what happens on the model.
Guessing the field being nullable is there just to satisfy passing an empty value through the form. Can't really change it there due to BC, but it would be nice to clean those in the future